backend/inspirehep/indexer/cli.py

```diff
@@ -391,6 +391,7 @@ def _put_files_pipeline():
                 {
                     "foreach": {
                         "field": "documents",
+                        "ignore_missing": True,
```

Comment:
is the config the same on tests and here? could we just reuse it instead of modifying it twice?